tag-version/action.yml

```diff
@@ -0,0 +1,58 @@
+name: Bump Version
+description: Bump the version and push if not a dry-run
+inputs:
+  version:
+    description: The new version
+    required: true
+  tag_template:
+    description: The template for the git tag
+    default: "${VERSION}"
+  tag_message_template:
+    description: The template for the git tag message
+    default: "BUMP ${VERSION}"
+  dry_run:
```

Comment:
Similar to the above, how about naming this `push_tag`?

Reply:
Done